### PR TITLE
Filled in 1.6a to print fields and first tweets

### PR DESCRIPTION
**Pull request by jjjjesse**
Unsure if I've misunderstood the  

> Write your code to print out all fields of the first tweet

### CODE REVIEW

File: group2_assignment2_solution.ipynb [hunk 1]
```diff
@@ -427,16 +452,20 @@
     "# INSERT YOUR CODE HERE\n",
     "\n",
     "\n",
-    "print()\n",
+    "print(k1_tweets[0].keys())\n",
+    "# ------Unsure if this is correct------\n",
+    "\n",
     "'''\n",
```
Comment (afestein) [inline group2_assignment2_solution.ipynb]: The questions are pretty badly worded but I think you have understood it correctly, we could probably do a loop I guess:
```
for key in k1_tweets[0].keys():
    print(key)
```

Comment (afestein) [reply group2_assignment2_solution.ipynb]: Same same but different 🤷‍♂️

File: group2_assignment2_solution.ipynb [hunk 2]
```diff
@@ -427,16 +452,20 @@
...
     "Write your code to print out the text of the first  tweet collected for each keyword.\n",
     "'''\n",
     "\n",
     "print(\"\\nThe text of the first tweet for \\\"{}\\\":\\n\".format(keywords[0]))\n",
     "# INSERT YOUR CODE HERE\n",
+    "print(k1_tweets[0]['text'])\n",
     "\n",
     "print('\\nThe text of the first tweet for \\\"{}\\\":\\n'.format(keywords[1]))\n",
     "# INSERT YOUR CODE HERE\n",
+    "print(k1_tweets[0][''])\n",
```
Comment (afestein) [inline group2_assignment2_solution.ipynb]: Shouldn't this one be `print(k2_tweets[0]['text'])`?

**Review by afestein — DISMISSED** [review]

File: group2_assignment2_solution.ipynb [hunk 3]
```diff
@@ -427,16 +452,20 @@
...
     "\n",
     "print('\\nThe text of the first tweet for \\\"{}\\\":\\n'.format(keywords[2]))\n",
     "# INSERT YOUR CODE HERE"
```
Comment (afestein) [inline group2_assignment2_solution.ipynb]: And looks like we need the third one `print(k3_tweets[0]['text'])` here.

File: group2_assignment2_solution.ipynb [hunk 4]
```diff
@@ -426,20 +443,23 @@
     "'''\n",
     "# INSERT YOUR CODE HERE\n",
     "\n",
+    "print(k1_tweets[0].keys())\n",
```
Comment (afestein) [inline group2_assignment2_solution.ipynb:446]: I'm happy with this, probably no need to loop through them on second thought. The purpose is to examine the keys and we can do that easily with this method 👍

File: group2_assignment2_solution.ipynb (hunk 3 above)
Comment (jjjjesse) [reply group2_assignment2_solution.ipynb]: Yeah sorry, looks like I didn't save properly before committing. My bad

**Comment by jjjjesse** [comment]
Need a review here

**Comment by afestein** [comment]
Oh, did you see my review and commits @jjjjesse?

**Comment by jjjjesse** [comment]
@afestein yeah all ooked good. A cant merge this without a review though

**Review by afestein — APPROVED** [review]

**Comment by afestein** [comment]
Ah OK yeah sorry.

**Pull request merged**